staramr/detection/AMRDetectionFactory.py

```diff
@@ -29,9 +30,9 @@ def build(self, plasmidfinder_database, resfinder_database, blast_handler, point
         """
 
         if include_resistances:
-            return AMRDetectionResistance(resfinder_database, ARGDrugTableResfinder(), blast_handler,
-                                          ARGDrugTablePointfinder(), pointfinder_database, include_negatives,
-                                          output_dir=output_dir, genes_to_exclude=genes_to_exclude,
+            return AMRDetectionResistance(resfinder_database, ARGDrugTableResfinder(), CGEDrugTableResfinder(),
```

Comment:
What you can do to handle the database location is change this to `CGEDrugTableResfinder(phenotypes_file)`, where you pass `phenotypes_file` into the CGE class.

The `phenotypes_file` location can be obtained from `resfinder_database.get_phenotypes_file()`, which you implement.

`resfinder_database` is an instance of this class https://github.com/phac-nml/staramr/blob/development/staramr/blast/resfinder/ResfinderBlastDatabase.py, which inherits from this abstract class https://github.com/phac-nml/staramr/blob/d050a21694f42a33045c5b03c7a626bf0810bcdf/staramr/blast/AbstractBlastDatabase.py#L10 which knows about the appropriate path to the resfinder/pointfinder database.

So, implement a method in https://github.com/phac-nml/staramr/blob/development/staramr/blast/resfinder/ResfinderBlastDatabase.py that is `get_phenotypes_file()` and uses `os.path.join(self.database_dir, 'phenotypes.txt')` to get the file.

Reply:
Implemented in [5cdf84f](https://github.com/phac-nml/staramr/pull/167/commits/5cdf84fa76be27d20e2b6449e803de28d9647e8c).

Should be working now.

Reply:
Looks great, thanks so much.